velox/functions/prestosql/aggregates/tests/PrestoHasherTest.cpp

```diff
@@ -356,4 +358,48 @@ TEST_F(PrestoHasherTest, wrongVectorType) {
   ASSERT_ANY_THROW(hasher.hash(vector, rows, hashes));
 }
 
+TEST_F(PrestoHasherTest, timestampWithTimezone) {
+  const auto toUnixtimeWithTimeZone =
+      [&](const std::vector<std::optional<std::pair<int64_t, std::string>>>&
+              timestampWithTimeZones) {
+        std::vector<std::optional<int64_t>> timestamps;
+        std::vector<std::optional<int16_t>> timeZoneIds;
+        auto size = timestampWithTimeZones.size();
+        timestamps.reserve(size);
+        timeZoneIds.reserve(size);
+
+        for (auto timestampWithTz : timestampWithTimeZones) {
+          if (timestampWithTz.has_value()) {
+            auto timestamp = timestampWithTz.value().first;
+            auto tz = timestampWithTz.value().second;
+            const int16_t tzid =
+                strcmp(tz.c_str(), "+00:00") == 0 ? 0 : util::getTimeZoneID(tz);
+            timeZoneIds.push_back(tzid);
+            timestamps.push_back(timestamp);
+          } else {
+            timeZoneIds.push_back(std::nullopt);
```

Comment:
Shouldn't this result in a null entry in the result vector? Currently, the result vector never has any nulls.